support/index.markdown

```diff
@@ -12,11 +12,11 @@ title: Support
 
   * First, check out the list of [Frequently Asked Questions](/documentation/faqs) to see if your question has been answered already.
```

Comment:
This is somewhat unrelated, but I feel like there aren't that many useful FAQs on that list. Perhaps this section should be moved to the bottom of this page